projects/ion/src/lib/popover/component/popover.component.spec.ts

```diff
@@ -0,0 +1,121 @@
+  describe('Check default fields', () => {
+    beforeEach(async () => {
+      await sut();
+    });
+
+    it('should render component with message', async () => {
+      expect(
+        screen.getByText(defaultProps.ionPopoverTitle)
+      ).toBeInTheDocument();
+    });
+
+    it('should render component with description', async () => {
+      expect(screen.getByText(defaultProps.ionPopoverBody)).toBeInTheDocument();
+    });
+  });
+
+  describe('with actions', () => {
+    beforeEach(async () => {
+      await sut({
+        ...defaultProps,
+        ionPopoverActions: [{ label: 'action 1' }, { label: 'action 2' }],
+      });
+    });
+
+    it.each(['action 1', 'action 2'])(
+      'should render button with default text %s',
+      async (textBtn: string) => {
+        expect(screen.getByText(textBtn)).toBeInTheDocument();
+      }
+    );
+
+    it.each(['popover-action-1', 'popover-action-2'])(
+      'should render %s',
+      async (btnId: string) => {
+        expect(screen.getByTestId(btnId)).toBeInTheDocument();
+      }
+    );
+  });
+
+  describe('check the icons', () => {
+    beforeEach(async () => {
+      await sut({
+        ...defaultProps,
+        ionPopoverIcon: 'condominium',
+        ionPopoverIconClose: true,
+      });
+    });
+
+    it('should render component with icon close', async () => {
+      expect(screen.getByTestId('popover-icon-close')).toBeInTheDocument();
+    });
+
+    it('should render component with icon close', async () => {
+      expect(
+        document.getElementById('ion-icon-condominium')
+      ).toBeInTheDocument();
+    });
+  });
+
+  describe('check the arrows', () => {
+    it.each(['leftTop', 'topCenter'])(
+      'should render component with arrow $s',
+      async (arrow: PopoverPosition) => {
+        await sut({
+          ...defaultProps,
+          ionPopoverPosition: arrow,
+        });
+        const element = screen.getByTestId('ion-popover');
+        expect(element).toHaveClass(`sup-container-${arrow}`);
+      }
+    );
+
+    it.each(['leftBottom', 'bottomCenter'])(
+      'should render component with arrow in footer $s',
+      async (arrow: PopoverPosition) => {
+        await sut({
+          ...defaultProps,
+          ionPopoverPosition: arrow,
+        });
+        const element = screen.getByTestId('ion-popover');
+        expect(element).toHaveClass(`sup-container-${arrow}`);
+      }
+    );
+
+    describe('with arrow and actions', () => {
+      it.each(['leftBottom', 'bottomCenter'])(
+        'should render component with arrow in footer $s',
+        async (arrow: PopoverPosition) => {
+          await sut({
+            ...defaultProps,
+            ionPopoverActions: [{ label: 'action 1' }, { label: 'action 2' }],
+            ionPopoverPosition: arrow,
+          });
+          const element = screen.getByTestId('ion-popover');
+          expect(element).toHaveClass(`sup-container-${arrow}`);
+        }
+      );
+    });
```

Comment:
```suggestion
```

I don't get it what you're testing with this assertion, since you're just repeating the test above with action buttons, and the buttons don't actually matter for arrow positioning.